server/api/rounds.py

```diff
@@ -198,6 +198,12 @@ def cumulative_batch_results(election: Election) -> BatchTallies:
     }
 
 
+def round_sizes(election: Election) -> Dict[int, int]:
+    return {
+        rnd.round_num: len(list(rnd.sampled_ballot_draws)) for rnd in election.rounds
```

Comment:
If the round_num values are always just counting up, would it make it easier to just return a simple List[int]? Then we don't have to argue whether the first round is 0 or 1 ;)

Reply:
I like it being explicit here, for actually a similar reason - there is no "round 0".

Reply:
Also If you wanted to do this in one database query, you could. slightly more efficient, but not a big deal either way

```py
return dict(
  Round.query.filter_by(election_id=election.id)
  .join(SampledBallotDraw).group_by(Round.id)
  .values(Round.round_num, func.count(SampledBallotDraw.ticket_number))
)
```

Reply:
Yeah, I'll use the db query, just couldn't quite figure out how to make it work.